packages/protocol/contracts/L1/TaikoToken.sol

```diff
@@ -80,42 +63,6 @@ contract TaikoToken is
         _snapshot();
     }
 
-    /// @notice Pauses token transfers.
-    function pause() public onlyOwner {
-        _pause();
-    }
-
-    /// @notice Unpauses token transfers.
-    function unpause() public onlyOwner {
-        _unpause();
-    }
-
-    /// @notice Mints new tokens to the specified address.
-    /// @param to The address to receive the minted tokens.
-    /// @param amount The amount of tokens to mint.
-    function mint(
-        address to,
-        uint256 amount
-    )
-        public
-        onlyFromNamed2("erc20_vault", "taiko")
-    {
-        _mint(to, amount);
-    }
-
-    /// @notice Burns tokens from the specified address.
-    /// @param from The address to burn tokens from.
-    /// @param amount The amount of tokens to burn.
-    function burn(
```

Comment:
We should not remove it (IMO) but just let it burnable - if anyone wants to burn.. be it. (Otherwise it effects the ERC20Vault as commented below.)

Reply:
Who would call it then? I think it's OK not to have it. If we really need it, for example, when we have a DAO to burn some tokens, we can upgrade the token contract. But before that, this feature won't be used intentionally.

Reply:
> Who would call it then?

`ERC20Vault.sol `line 225 ? 🤔

Reply:
@dantaik